[debug] [2020-04-13T16:58:04.949Z] ----------------------------------------------------------------------
[debug] [2020-04-13T16:58:04.950Z] Command:       /usr/local/Cellar/node/13.11.0/bin/node /Users/ibrahimkurhan/.config/yarn/global/node_modules/.bin/firebase init hosting
[debug] [2020-04-13T16:58:04.951Z] CLI Version:   8.0.2
[debug] [2020-04-13T16:58:04.951Z] Platform:      darwin
[debug] [2020-04-13T16:58:04.951Z] Node Version:  v13.11.0
[debug] [2020-04-13T16:58:04.952Z] Time:          Mon Apr 13 2020 17:58:04 GMT+0100 (British Summer Time)
[debug] [2020-04-13T16:58:04.952Z] ----------------------------------------------------------------------
[debug] [2020-04-13T16:58:04.952Z] 
[debug] [2020-04-13T16:58:04.958Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-13T16:58:05.074Z] Error: Unexpected error determining execution environment: Invalid response from metadata service: incorrect Metadata-Flavor header.
    at metadataAccessor (/Users/ibrahimkurhan/.config/yarn/global/node_modules/gcp-metadata/build/src/index.js:60:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Object.isAvailable (/Users/ibrahimkurhan/.config/yarn/global/node_modules/gcp-metadata/build/src/index.js:161:9)
    at async GoogleAuth._checkIsGCE (/Users/ibrahimkurhan/.config/yarn/global/node_modules/google-auth-library/build/src/auth/googleauth.js:176:31)
    at async GoogleAuth.getApplicationDefaultAsync (/Users/ibrahimkurhan/.config/yarn/global/node_modules/google-auth-library/build/src/auth/googleauth.js:152:21)
    at async GoogleAuth.getClient (/Users/ibrahimkurhan/.config/yarn/global/node_modules/google-auth-library/build/src/auth/googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (/Users/ibrahimkurhan/.config/yarn/global/node_modules/google-auth-library/build/src/auth/googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2020-04-13T16:58:13.582Z] ----------------------------------------------------------------------
[debug] [2020-04-13T16:58:13.584Z] Command:       /usr/local/Cellar/node/13.11.0/bin/node /Users/ibrahimkurhan/.config/yarn/global/node_modules/.bin/firebase init
[debug] [2020-04-13T16:58:13.584Z] CLI Version:   8.0.2
[debug] [2020-04-13T16:58:13.584Z] Platform:      darwin
[debug] [2020-04-13T16:58:13.584Z] Node Version:  v13.11.0
[debug] [2020-04-13T16:58:13.585Z] Time:          Mon Apr 13 2020 17:58:13 GMT+0100 (British Summer Time)
[debug] [2020-04-13T16:58:13.585Z] ----------------------------------------------------------------------
[debug] [2020-04-13T16:58:13.585Z] 
[debug] [2020-04-13T16:58:13.592Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-13T16:58:14.834Z] Error: Unexpected error determining execution environment: Invalid response from metadata service: incorrect Metadata-Flavor header.
    at metadataAccessor (/Users/ibrahimkurhan/.config/yarn/global/node_modules/gcp-metadata/build/src/index.js:60:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Object.isAvailable (/Users/ibrahimkurhan/.config/yarn/global/node_modules/gcp-metadata/build/src/index.js:161:9)
    at async GoogleAuth._checkIsGCE (/Users/ibrahimkurhan/.config/yarn/global/node_modules/google-auth-library/build/src/auth/googleauth.js:176:31)
    at async GoogleAuth.getApplicationDefaultAsync (/Users/ibrahimkurhan/.config/yarn/global/node_modules/google-auth-library/build/src/auth/googleauth.js:152:21)
    at async GoogleAuth.getClient (/Users/ibrahimkurhan/.config/yarn/global/node_modules/google-auth-library/build/src/auth/googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (/Users/ibrahimkurhan/.config/yarn/global/node_modules/google-auth-library/build/src/auth/googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2020-04-13T16:58:32.668Z] ----------------------------------------------------------------------
[debug] [2020-04-13T16:58:32.670Z] Command:       /usr/local/Cellar/node/13.11.0/bin/node /Users/ibrahimkurhan/.config/yarn/global/node_modules/.bin/firebase login
[debug] [2020-04-13T16:58:32.670Z] CLI Version:   8.0.2
[debug] [2020-04-13T16:58:32.670Z] Platform:      darwin
[debug] [2020-04-13T16:58:32.670Z] Node Version:  v13.11.0
[debug] [2020-04-13T16:58:32.671Z] Time:          Mon Apr 13 2020 17:58:32 GMT+0100 (British Summer Time)
[debug] [2020-04-13T16:58:32.671Z] ----------------------------------------------------------------------
[debug] [2020-04-13T16:58:32.671Z] 
[info] i  Firebase optionally collects CLI usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.

[info] i  To change your data collection preference at any time, run `firebase logout` and log in again.
[info] 
[info] Visit this URL on this device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=903154845&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
